src/modals/GeolocationModal.jsx

```diff
@@ -3,40 +3,47 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Modal from 'src/components/ui/Modal';
 import { listen } from 'src/libs/customEvents';
+import { Button, CloseButton } from 'src/components/ui';
+import classnames from 'classnames';
 
-const GeolocationModal = ({ status, onClose }) => {
+const GeolocationModal = ({ status, onClose, onClick }) => {
   /* eslint-disable max-len */
   const statuses = {
     PENDING: {
-      title: _('Geolocation with privacy!', 'geolocation'),
-      text: _('You need to know where you are? Please allow your application to access your location so that it can tell you. As stated in our {privacyPolicyLink} privacy policy {closeTag}, Qwant will not store your geolocation information. We don\'t want to know your whereabouts.',
+      title: _('Enable your geolocation for better directions', 'geolocation'),
+      text: _('Always respecting your privacy.<br>As stated in {privacyPolicyLink}our privacy policy{closeTag}, we don\'t store your information because we don\'t want to know your whereabouts.',
         'geolocation', {
           privacyPolicyLink: '<a target="_blank" rel="noopener noreferrer" href="https://about.qwant.com/legal/privacy">',
           closeTag: '</a>',
         }
       ),
-      button: _('Ok, I\'ve got it.', 'geolocation'),
+      button: _('Ok, I\'ve got it', 'geolocation'),
+      className: 'modal__maps__pending',
     },
     DENIED: {
       title: _('Geolocation not available.', 'geolocation'),
       text: _('You have not allowed Qwant Maps (or your device) to access your location.', 'geolocation'),
       button: _('Ok', 'geolocation'),
+      className: 'modal__maps__denied',
     },
     NOT_ACTIVATED: {
       title: _('Geolocation not available.', 'geolocation'),
       text: _('Qwant Maps cannot determine your position. Check that location services are enabled on your system.'),
       button: _('Ok', 'geolocation'),
+      className: 'modal__maps__not-activated',
     },
   };
   /* eslint-enable max-len */
 
-  const { title, text, button } = statuses[status];
+  const { title, text, button, className } = statuses[status];
   return <Modal onClose={onClose}>
-    <div className="modal__maps">
-      <i className="icon-x modal__close" onClick={onClose} />
-      <h2 className="modal__title">{title}</h2>
-      <div className="modal__subtitle" dangerouslySetInnerHTML={{ __html: text }} />
-      <button className="modal__button" onClick={onClose}>{button}</button>
+    <div className={classnames('modal__maps', className)}>
+      <CloseButton onClick={onClose} />
+      <div className="modal__maps__content">
+        <h2 className="modal__title u-text--smallTitle">{title}</h2>
+        <div className="modal__subtitle u-text--subtitle" dangerouslySetInnerHTML={{ __html: text }} />
```

Comment:
According to the linter, this line is too long.

Reply:
sorry, my local linter was misbehaving